docs/src/index.md

```diff
@@ -0,0 +1,4 @@
+# DocsScraper
+
+## Documentation
+
```

Comment:
This is empty! Add some usage examples and autodocs

eg,
````julia
# Reference

```@index
```

```@autodocs
Modules = [DocsScraper]
```
````